examples/starknet_multiple_contracts/src/balance.cairo

```diff
@@ -1,18 +1,19 @@
-#[contract]
+#[starknet::contract]
 mod Balance {
+    #[storage]
     struct Storage {
-        balance: felt252,
+        balance: felt252, 
     }
 
     // Increases the balance by the given amount.
     #[external]
-    fn increase_balance(amount: felt252) {
-        balance::write(balance::read() + amount);
+    fn increase_balance(ref self: ContractState, amount: felt252) {
+        self.balance.write(self.balance.read() + amount);
     }
 
     // Returns the current balance.
     #[view]
-    fn get_balance() -> felt252 {
-        balance::read()
+    fn get_balance(ref self: ContractState) -> felt252 {
+        self.balance.read()
     }
 }
```

Comment:
I think `#[view]` tag is no longer relevant. This method should just accept `self: @ContractState` - immutable snapshot of state instead of mutable reference.